### ISSUE TITLE
[Perf Refactor] Optimize the settings screen UI.

### ISSUE THREAD

**Issue by aakash1313** [covidsafe/App-Android#63]
While trying to add an entry for the language selector in the app, found that this we are using multiple recycler views, we can achieve the same by using less or single recylcer, need to add have before adding an entry for language settings. This will make it easy to support more entries in future.

**Comment by justinklchan** [comment]
Yes this is a good fix to make. If you can just use a single recycler view to represent settings with a Switch, a Dropdown, and a Link. That would be good. Keep in mind that the recycler view used for the settings for switches is also used in the Permissions fragment during onboarding. So make sure all these dependencies still work before pushing.
